drivers/espi/Kconfig

```diff
@@ -22,8 +22,23 @@ config ESPI_SLAVE
 	help
 	  Enables eSPI driver in slave mode.
 
+config ESPI_AUTOMATIC_WARNING_ACKNOWLEDGE
+	bool "ESPI slave driver"
+	default y
+	help
+	  Enable automatic acknowledge from eSPI slave towards eSPI host
+	  whenever it receives suspend or reset warning.
+	  If this is disabled, it means the app wants to be give the opportunity
+	  to prepare for either HOST suspend or reset.
+
 config ESPI_INIT_PRIORITY
-	int "IRQ Priority for ESPI Controller"
+	int "Driver initialization priority for ESPI Controller"
+	default 3
+	help
+	  IRQ Priority for ESPI Controller.
+
+config ESPI_IRQ_PRIORITY
```

Comment:
This should come from a DT macro. Remove the entry from Kconfig

interrupts = <11 0>, <15 0>, <7 0>;

The second parameter is the priority

And this are the autogenerated macros
#define DT_MICROCHIP_XEC_ESPI_400F3400_IRQ_0_PRIORITY 0

#define DT_MICROCHIP_XEC_ESPI_400F3400_IRQ_1_PRIORITY 0 

#define DT_MICROCHIP_XEC_ESPI_400F3400_IRQ_2_PRIORITY 0

You can add an entries in the fixup to hide the address

DT_ESPI_MCHP_XEC_0_IRQ_PRIORITY
DT_ESPI_MCHP_XEC_1_IRQ_PRIORITY
DT_ESPI_MCHP_XEC_2_IRQ_PRIORITY